src/imports.js

```diff
@@ -22,7 +23,7 @@ export class Component extends _React_.Component {
    * @returns {void}
    */
   dispatchEvent(name, detail, bubbles = true) {
-    const event = new CustomEvent(name, {detail, bubbles});
+    const event = new window.CustomEvent(name, {detail, bubbles});
```

Comment:
Why did you add the window here?

Reply:
backwards compatibility.  there isn't always a CustomEvents global (and / or environments dont always default to checking the `window` object for globals; node test environments break)

<img width="627" alt="screen shot 2018-05-16 at 14 38 16" src="https://user-images.githubusercontent.com/1480168/40117291-cdb33b30-5916-11e8-98cc-de27d9799675.png">

Reply:
I never had this error before so there must be a change in meanwhile. Even if there is the window object now the error must be kind of smae because CustomEvent will be undefined then